geoprocessing/src/modules/protected-areas/protected-areas.module.ts

```diff
@@ -5,11 +5,16 @@ import { ProtectedAreasController } from './protected-areas.controller';
 import { ProtectedAreasService } from './protected-areas.service';
 import { TileModule } from 'src/modules/tile/tile.module';
 import { ProtectedArea } from 'src/modules/protected-areas/protected-areas.geo.entity';
+import { ProtectedAreaWorkerModule } from './worker/protected-area-worker.module';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([ProtectedArea]), TileModule],
+  imports: [
+    TypeOrmModule.forFeature([ProtectedArea]),
+    TileModule,
+    ProtectedAreaWorkerModule,
+  ],
   providers: [ProtectedAreasService],
   controllers: [ProtectedAreasController],
-  exports: [ProtectedAreasService],
+  exports: [ProtectedAreasService, TypeOrmModule],
```

Comment:
@Dyostiq correct me if I am wrong, but otherwise, while using `ProtectedAreasService` , in target module, TypeOrm raised that entity is missing (i.e. would require to manually load .forFeature)

Reply:
No, it shouldn't, because the provider is in this module, not in modules that imports this module and use the service.